src/scenes/AcceleratorRouter/ParticipantProjects/Scoring/EditView.tsx

```diff
@@ -83,7 +96,13 @@ const ScorecardEditView = () => {
   }, [updateStatus, goToScorecardView]);
 
   return (
-    <Page title={`Edit Scoring for project ${projectName}`} crumbs={crumbs} hierarchicalCrumbs={false}>
+    <PageWithModuleTimeline
+      title={`Edit Scoring for project ${project?.name ?? ''}`}
```

Comment:
We should localize this title (I know it's not form these changes)

Reply:
Looks like it is not actually used! Removing the title all together!